### PR TITLE
make tick width reactive

### LINKED ISSUE #4129: y-axes for stacked plots do not line up
#### Summary

Space is collapsing down to the size of each y-axis value set. Different y-axis value widths cause plots to not be lined up vertically.

#### Expected vs Current Behavior

The y-axes should adjust y-axis value width so that the left-most part of all plots line up vertically.

#### Impact Check List

- [ ] Data loss or misrepresented data?
- [x] Regression? Did this used to work or has it always been broken?
- [ ] Is there a workaround available?
- [x] Does this impact a critical component?
- [x] Is this just a visual bug?

#### Steps to Reproduce

1. Create a stacked plot
2. Observe whether the y-axes line up vertically with one another
2. Adjust the plot data/type as needed so that various y-axis value widths are present. ie. 5.3 and 64.0005

#### Environment
* Open MCT Version: 1.7.6
* Deployment Type: 
* OS:
* Browser:

#### Additional Information

image

### PR DESCRIPTION
**Pull request by davetsay**
## Overview
Closes #4129 
### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?

### Author Checklist

* [] Changes address original issue?
* [ ] Unit tests included and/or updated with changes?
* [] Command line build passes?
* [] Has this been smoke tested?
* [ ] Testing instructions included in associated issue?

### CODE REVIEW

File: src/plugins/plot/stackedPlot/StackedPlotItem.vue [hunk 1]
```diff
@@ -120,7 +120,9 @@ export default {
             });
         },
         onTickWidthChange() {
-            this.$emit('plotTickWidth', ...arguments);
+            const id = this.openmct.objects.makeKeyString(this.object.identifier);
+            const args = [...arguments, id];
+            this.$emit('plotTickWidth', ...args);
```
Comment (davetsay) [inline src/plugins/plot/stackedPlot/StackedPlotItem.vue:125]: probably either get all arguments from `mctplot.vue`, or only accept tick width from mctplot.vue and add id

**Review by shefalijoshi — APPROVED** [review]
Looks good.

**Comment by nikhilmandlik** [comment]
please change base branch from master to 1.7.8

**Comment by shefalijoshi** [comment]
Closing as this has been merged into 1.7.8

**Pull request closed without merging**